waffle-ens/src/EnsBuilder.ts

```diff
@@ -38,33 +42,56 @@ export class ENSBuilder {
     await this.ens.setSubnodeOwner(HashZero, utils.id(domain), this.registrars[domain].address);
   }
 
-  async createSubDomain(domain: string) {
+  async createSubDomainNotRecursive(domain: string) {
```

Comment:
`createSubDomainNotRecursive` -> `createSubDomainNonRecursive`
`*Not*` -> `*Non*`
#non